themes/ddbasic/gulpfile.js

```diff
@@ -81,13 +81,38 @@ gulp.task('sass', 'Process SCSS using libsass',
   }
 );
 
+/**
+ * Usage:
+ * gulp ksssass
+ *
+ * Precompile all sass files into one single bundle.css file in the sass_css folder.
+ * This all in one file is used by kss for the style sheet.
+ *
+ */
+gulp.task('ksssass', 'Process SCSS into one css file using libsass',
+  function () {
+    gulp.src(sassPath)
+      .pipe(sass({outputStyle: 'compact'})
+        .on('error', sass.logError))
+      // Save verbose output for testing purposes.
+      .pipe(gulp.dest('./sass_css_verbose_output'))
+      // Minify the css and save it.
+      .pipe(cleanCSS())
+      .pipe(concat('bundle.css'))
+      .pipe(gulp.dest('./sass_css'));
```

Comment:
This code to generate `bundle.css` seems to be the same as L126-129. Are they both needed?

Reply:
The two tasks have been joined. ksssass is now deleted.